python/sglang/srt/models/llama.py

```diff
@@ -350,8 +351,17 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                 if name.endswith(".bias") and name not in params_dict:
                     continue
                 param = params_dict[name]
-                weight_loader = param.weight_loader
-                weight_loader(param, loaded_weight, shard_id)
+                if hasattr(param, "weight_loader"):
```

Comment:
why is this "if" needed?

Reply:
it seems that the `param` can be quantized already at this point, this is to skip the loading if it's quantized already, I can debug more to identify what weight this is

Reply:
fixed now in the new update, I was not clear about what this is doing before